packages/preview/truthfy/0.1.0/README.md

```diff
@@ -0,0 +1,24 @@
+# truthfy
+Make an empty or filled truth table in Typst
+
+# Functions
+
+```
+generate_empty(info: array[math_block], data: array[str]): table
```

Comment:
Just wanted to note that the idiomatic Typst case would be `generate-empty` (i.e. kebab-case). It's up to you whether you want to change it to that or not though.

Reply:
Done!